applications/GeoMechanicsApplication/custom_utilities/solving_strategy_factory.hpp

```diff
@@ -21,91 +21,101 @@
 #include "scheme_factory.hpp"
 #include "convergence_criteria_factory.hpp"
 #include "builder_and_solver_factory.hpp"
+#include "solving_strategies/strategies/line_search_strategy.h"
```

Comment:
I think it would make sense to group this with the other strategy-related includes

Reply:
done